borsh-derive/Cargo.toml

```diff
@@ -17,13 +17,20 @@ Binary Object Representation Serializer for Hashing
 proc-macro = true
 
 [dependencies]
-borsh-derive-internal = { path = "../borsh-derive-internal", version = "0.11.0" }
```

Comment:
add `refactor!:` prefix to pr's title so that `git-cliff` can pick it up for changelog